src/webapp/components/tag/Tag.tsx

```diff
@@ -0,0 +1,39 @@
+import styled, { css } from "styled-components";
+import customTheme from "$/webapp/pages/app/themes/customTheme";
+import { memo } from "react";
```

Comment:
No need to change, but normally import the whole React object and then we use `React.memo` in the code. The advantage is to have a shorter import.